onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider.cc

```diff
@@ -1705,1838 +1702,1830 @@ SubGraphCollection_t NvExecutionProvider::GetSupportedList(SubGraphCollection_t
-                               "https://onnxruntime.ai/docs/tutorials/web/env-flags-and-session-options.html#freedimensionoverrides.";
-      trt_profiles.push_back(trt_builder->createOptimizationProfile());
-    }
-  }
-  if (has_dynamic_shape) {
-    // Iterate all input tensors to check dynamic shape
-    for (unsigned int i = 0, end = num_inputs; i < end; ++i) {
-      auto input = trt_network->getInput(i);
-      const std::string& input_name = input->getName();
-      nvinfer1::Dims dims = input->getDimensions();
-
-      // Apply explicit optimization profiles provided by user
-      bool apply_profile = false;
-      bool tensor_has_profile = profile_min_shapes_.find(input_name) != profile_min_shapes_.end() &&
-                                profile_opt_shapes_.find(input_name) != profile_opt_shapes_.end() &&
-                                profile_max_shapes_.find(input_name) != profile_max_shapes_.end();
-      if (has_explicit_profile && tensor_has_profile) {
-        apply_profile = ApplyProfileShapesFromProviderOptions(trt_profiles, input, profile_min_shapes_, profile_max_shapes_, profile_opt_shapes_, input_explicit_shape_ranges, cuda_graph_enable_);
-      } else {
-        LOGS_DEFAULT(INFO) << "[NvTensorRTRTX EP] Creating implicit profile for tensor " << input_name;
-        profile_min_shapes_[input_name] = std::vector<std::vector<int64_t>>{{}};
-        profile_min_shapes_[input_name][0].resize(dims.nbDims);
-        profile_opt_shapes_[input_name] = std::vector<std::vector<int64_t>>{{}};
-        profile_opt_shapes_[input_name][0].resize(dims.nbDims);
-        profile_max_shapes_[input_name] = std::vector<std::vector<int64_t>>{{}};
-        profile_max_shapes_[input_name][0].resize(dims.nbDims);
-        for (int idx_dim = 0; idx_dim < dims.nbDims; ++idx_dim) {
-          auto dim_value = dims.d[idx_dim];
-          if (dim_value == -1) {
-            has_implicit_profile = true;
-            // TODO(maximilianm) this is needed until we have a wildcard in the API to support dynamic shapes
-            profile_min_shapes_[input_name][0][idx_dim] = 0;
-            // TODO(maximilianm) This can be buggy since shape inference can failt with 1 being used as optimal shape
-            //        [2025-04-04 15:41:58   ERROR] IBuilder::buildSerializedNetwork: Error Code 4: Internal Error (kOPT values for profile 0 violate shape constraints:
-            //        /conv1/Conv: spatial dimension of convolution/deconvolution output cannot be negative (build-time output dimension of axis 2 is
-            //        (+ (CEIL_DIV (+ h -6) 2) 1)) Condition '<' violated: 2 >= 1.)
-            profile_opt_shapes_[input_name][0][idx_dim] = 1;
-            profile_max_shapes_[input_name][0][idx_dim] = std::numeric_limits<int16_t>::max();
+    // exclude weights if external
+    std::vector<TensorrtUserWeights> userWeights;
+    if (use_external_data_initializer_) {
```

Comment:
Same here.
Missing "`}`" for this if block.

Reply:
@chilo-ms, can you add a CI pipeline for the rtx ep?

Reply:
i had a [PR ](https://github.com/microsoft/onnxruntime/pull/26343)for adding Windows CI for RTX EP previously, but it got segfault for the 'LargeModel' unit test that parses large model and it's hard for us to debug due to we don't have A10 GPU to access to.

Nvidia is also aware of this issue. 
Let me see how to make this work, probably by commenting out the unit test for now.